lib/auth/grpcserver.go

```diff
@@ -2500,6 +2498,8 @@ func validateUserSingleUseCertRequest(ctx context.Context, actx *grpcContext, re
 		}
 	case proto.UserCertsRequest_All:
 		return trace.BadParameter("must specify a concrete Usage in UserCertsRequest, one of SSH, Kubernetes or Database")
+	case proto.UserCertsRequest_App:
```

Comment:
this change is unrelated to device trust, right?

Reply:
Correct, it's just a custom error message, otherwise it falls to the default of "unknown certificate Usage".